app/bundles/DashboardBundle/Event/WidgetDetailEvent.php

```diff
@@ -149,26 +175,39 @@ public function getTemplate()
 
     /**
      * Set the widget template data.
+     *
+     * @throws \Psr\Cache\InvalidArgumentException
      */
-    public function setTemplateData(array $templateData, $skipCache = false): void
+    public function setTemplateData(array $templateData, ?bool $skipCache = false): void
```

Comment:
Please add the new param type to a method comment. Changing a method signature is considered a BC break. We'll have to add the real types for M6.